### DISCUSSION TITLE
pnpm install  ，并么有安装@@/requestRecordMock这个包，配置都是taobao 源，什么原因？

### DISCUSSION THREAD

**Discussion by huangjihua** [umijs/umi#10254] [Q&A]
![image](https://user-images.githubusercontent.com/3156292/211446439-7da443c6-aca3-4ce7-8995-6ab270edadd7.png)

**Comment by fz6m** [comment]
umi4 antd pro 吗，来这里 https://github.com/umijs/umi-presets-pro/issues/11 反馈一下。

**Reply by huangjihua** [reply]
> umi4 antd pro 吗，来这里 [umijs/umi-presets-pro#11](https://github.com/umijs/umi-presets-pro/issues/11) 反馈一下。

是的
